### PR TITLE
remove the 102032 error message

### PR DESCRIPTION
**Pull request by Peter0x44**

### CODE REVIEW

**Review by Artuto — COMMENTED** [review]
Changes

File: src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java [hunk 1]
```diff
@@ -242,7 +242,7 @@ protected void execute(CommandEvent event)
             List<String> args = parseArgs(REMOVE_PATTERN, event.getArgs());
             if(args.size() < 2)
             {
-                event.replyError("Wrong format! Correct one is `" + gs.getFirstPrefix() + "code remove <type> <name>");
+                event.replyError("Wrong format! Correct one is ``" + gs.getFirstPrefix() + "code remove <type> <name>``");
```
Comment (Artuto) [inline src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java]: Why double "`"

File: src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java [hunk 2]
```diff
@@ -257,7 +257,6 @@ private void success(@NotNull CommandEvent event, @NotNull Response response)
     private final Map<Integer, String> codeNotes = new HashMap<Integer, String>()
     {{
         put(101409, "If you are getting this problem while doing something with Wii Mail, check if you patched the nwc24msg.cfg correctly. https://bit.ly/2QUrsyD");
-        put(102032, "The IOS the app/game uses is not patched for RiiConnect24. Try sending a message again but do it quickly, you need to do it in less than a minute. We hope to improve Wii Mail to stop getting this error.");
```
Comment (Artuto) [inline src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java:260]: Not sure.... I doubt that error will never seen again

Comment (Peter0x44) [reply src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java:260]: well, I guess we can remove the last part about improving wii mail at the very least

File: src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java (hunk 1 above)
Comment (Peter0x44) [reply src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java]: I guess that does the same thing, not that it's a big deal

Comment (Artuto) [reply src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java]: Remove that, Discord is prone to breaking stuff easily

File: src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java (hunk 2 above)
Comment (Artuto) [reply src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java:260]: Ok, remove the "Try sending a message...." part

**Review by Artuto — APPROVED** [review]
lgtm

**Pull request merged**